python/python_fastapi_server/routers/edr.py

```diff
@@ -262,11 +273,10 @@ async def get_collection_position(
 }
 
 
-@aiocache.cached(ttl=60, serializer=PickleSerializer())
 async def get_collectioninfo_for_id(
```

Comment:
Function name is inconsistent with comment on line 284. Rename either. :-)

Reply:
Done